com.rustdesk.RustDesk.json

```diff
@@ -81,9 +84,12 @@
           "sha256": "f7d675ed2160c05b2206b6b7e50a2d369b6ed7f63abe0e89b9f38f9a29793fb4",
           "dest-filename": "rustdesk.deb",
           "x-checker-data": {
-            "type": "anitya",
-            "project-id": "309593",
-            "url-template": "https://github.com/rustdesk/rustdesk/releases/download/$version/rustdesk-$version-x86_64.deb"
+            "type": "json",
+            "url": "https://api.github.com/repos/rustdesk/rustdesk/releases/latest",
```

Comment:
I'm not sure, but is it okay for it to always be the latest version in "url" field? For example, if there is a new version 1.3.2, would it somehow affect the 1.3.1 installation with this config? Sorry for maybe irrelevant questions, I just want to be sure that these changes are ok and I have a bit of lack of experience here. Anyway, I tested it locally with 1.3.0 (the latest version now is 1.3.1) and it seems to build fine.

Reply:
>For example, if there is a new version 1.3.2, would it somehow affect the 1.3.1 installation with this config?

Not at all. That `url` property under `x-checker-data` is only used by the bot to check what's the current version of RustDesk.

If the bot determines the current version to be `1.3.2`, it will open a pull request on this repository updating the `url` source exactly [here](https://github.com/flathub/com.rustdesk.RustDesk/blob/8731340a620274d048eedd2384954e6f3baef850/com.rustdesk.RustDesk.json#L80) and [here](https://github.com/flathub/com.rustdesk.RustDesk/blob/8731340a620274d048eedd2384954e6f3baef850/com.rustdesk.RustDesk.json#L66). And it will also add a new `<release>` tag [here](https://github.com/flathub/com.rustdesk.RustDesk/blob/8731340a620274d048eedd2384954e6f3baef850/com.rustdesk.RustDesk.metainfo.xml#L59). Notice how we're not talking about the `url` property under `x-checker-data` here.

Anyway, I think if you see a [practical example](https://github.com/flathub/org.qownnotes.QOwnNotes/pull/301) of how to bot works, you'll understand it better.